Gemfile

```diff
@@ -18,12 +18,19 @@ gem 'bundler', '~> 2'
 gem 'rake', '~> 13'
 gem 'yard', '~> 0.9'
 
-gem 'rubocop', '~> 1.28'
-gem 'rubocop-rake', '~> 0.6'
-gem 'rubocop-rspec', '~> 2.10'
+if RUBY_VERSION.start_with?('2.4')
```

Comment:
This works but is pretty flaky. For next time or as a quick follow, use `Gem::Version`, see https://code.dblock.org/2020/07/16/comparing-version-numbers-in-ruby.html

A better idea is probably to downgrade the version of Rubocop to the last one that supports Ruby 2.4 (1.12?).